static/app/views/alerts/rules/issue/ticketRuleModal.tsx

```diff
@@ -178,16 +178,24 @@ class TicketRuleModal extends AbstractExternalIssueForm<Props, State> {
         const prevChoice = instance?.[field.name];
```

Comment:
Did you tests bot editing existing alert rule and creating a new one? This logic makes me think it's used in both places.

Reply:
yep! I've hand tested this for new, existing, and currently unsaved ones being edited as well.